src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.html

```diff
@@ -36,8 +36,11 @@
             </div>
             <div class="form-group">
                 <label>{{ 'artemisApp.attachmentUnit.createAttachmentUnit.file' | artemisTranslate }}*</label>
-                <small id="fileHelp" class="form-text text-muted">{{ 'artemisApp.attachmentUnit.createAttachmentUnit.fileFormat' | artemisTranslate }} </small>
-                <fa-icon [icon]="farQuestionCircle" [ngbTooltip]="fileLimitations"></fa-icon>
+                <small id="fileHelp" class="form-text text-muted">{{ 'artemisApp.attachmentUnit.createAttachmentUnit.fileLimitation' | artemisTranslate }} </small>
+                <fa-icon
+                    [icon]="faQuestionCircle"
+                    [ngbTooltip]="'artemisApp.attachmentUnit.createAttachmentUnit.fileLimitationTooltip' | artemisTranslate: { extensions: allowedFileExtensions }"
+                ></fa-icon>
                 <div class="input-group background-file">
                     <div class="custom-file overflow-ellipsis">
                         <input
```

Comment:
Adding [`accept`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept) attribute like the example below can further increase the usability by filtering out the ineligible file formats from the browser's file upload UI.  `allowedFileExtensions` property is almost in the correct format for `accept` but it needs dots before the extensions.
```
<input accept=".png,.jpg,.jpeg,.gif" ... >
```

Reply:
Good idea!